### DISCUSSION TITLE
How to use ConfuserEX

### DISCUSSION THREAD

**Discussion by ColeVan** [mkaring/ConfuserEx#381] [Q&A]
Total Noob here!

I'm desperately trying to figure out how to launch and use ConfuserEX GUI. I have the master repo pulled down and unzipped. I also have visual studio 2019 loaded up and ready to go. Do I build ConfuserEX, compile additional code, or execute something specific. Any help would be greatly appreciated.

**Comment by mkaring** [comment]
I recommend using the official release builds. Get yourself the latest release build [here](https://github.com/mkaring/ConfuserEx/releases/latest). After downloading and unzipping `ConfuserEx-GUI.zip` you'll find a `ConfuserEx.exe` executable.
This is your starting point and where you can start dragging the assemblies you want to protect in.
